src/app/server-cluster/testing/EmptyProvider.cpp

```diff
@@ -28,6 +28,31 @@ CHIP_ERROR EmptyProvider::Shutdown()
     return CHIP_NO_ERROR;
 }
 
+void EmptyProvider::SetNodeConfigurationListener(app::DataModel::NodeConfigurationListener * nodeConfigurationListener)
+{
+    return;
+}
+
+void EmptyProvider::NotifyNodeConfigurationListener()
+{
+    return;
+}
+
+CHIP_ERROR EmptyProvider::GetNodeDataModelConfiguration(app::DataModel::NodeDataModelConfiguration & nodeDataModelConfiguration)
+{
+    return CHIP_NO_ERROR;
```

Comment:
Doesn't it need to actually set some values?  Or at least return an error if it can't....

Reply:
Sorted in latest commit